src/components/appointments/Appointments.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
Same case here for array index. I am sure there is a workaround. What do you think?